decidim-proposals/app/commands/decidim/proposals/admin/create_proposal_note.rb

```diff
@@ -48,18 +50,17 @@ def create_proposal_note
           )
         end
 
-        def notify_admins_and_valuators
-          affected_users = Decidim::User.org_admins_except_me(form.current_user).all
-          affected_users += Decidim::Proposals::ValuationAssignment.includes(valuator_role: :user).where.not(id: form.current_user.id).where(proposal:).map(&:valuator)
+        def notify_mentioned
+          affected_users = mentioned_admins_or_valuators
+
+          return if affected_users.blank?
 
-          data = {
+          Decidim::EventsManager.publish(
             event: "decidim.events.proposals.admin.proposal_note_created",
```

Comment:
I think this notification is not clear. What I've received when I was mentioned is the following text: 

> Someone has left a note on the proposal XXX. Check it out at the admin panel. 

I had to double check the code to see if I was doing something wrong, because it wasn't just a note what I was receiving, but a mention. As we already have mentions in comments ("You have been mentioned in XXX by YYY @ZZZ" is what the mentioned participant receives), I was expecting to have a mention to the mention in the notification 😄 

This is what I was expecting: 

> You have been mentioned in a private note in XXX by YYY @ZZZ. Check it out at the admin panel. 

Can you create a new Event for mentions in the private notes with this text 🙏🏽 ?

Reply:
Added 2 different events texts for the same event which includes the author of the note